cristin-import/src/main/java/no/unit/nva/cristin/patcher/CristinPatcher.java

```diff
@@ -23,17 +29,44 @@ public static ParentAndChild updateChildPublication(ParentAndChild parentAndChil
         var parent = parentAndChild.getParentPublication();
         if (ChildParentInstanceComparator.isValidCombination(child, parent)) {
             var anthology = (Anthology) child.getEntityDescription().getReference().getPublicationContext();
-            anthology.setId(createPartOfUri(parentAndChild.getParentPublication()));
+            anthology.setId(createPublicationId(parentAndChild.getParentPublication()));
         } else {
-            throw new PublicationInstanceMismatchException(
+            throw new ChildPatchPublicationInstanceMismatchException(
                 String.format(CHILD_PARENT_REFERENCE_MISMATCH,
                               ChildParentInstanceComparator.getPublicationsInstanceName(child),
                               ChildParentInstanceComparator.getPublicationsInstanceName(parent)));
         }
         return parentAndChild;
     }
 
-    private static URI createPartOfUri(Publication parentPublication) {
-        return UriWrapper.fromUri(NVA_API_DOMAIN + PUBLICATION_PATH + "/" + parentPublication.getIdentifier()).getUri();
+    private static boolean isDegreePhd(Publication publication) {
+        return publication.getEntityDescription()
+                   .getReference()
+                   .getPublicationInstance() instanceof DegreePhd;
+    }
+
+    public static ParentAndChild updateParentPublication(ParentAndChild parentAndChild) {
+        return isDegreePhd(parentAndChild.getParentPublication())
+                   ? attemptToUpdateParentPublication(parentAndChild)
+                   : parentAndChild;
+    }
```

Comment:
I think we want conditional here in order to not to spam with a lot of error messages when patching types we do not support.